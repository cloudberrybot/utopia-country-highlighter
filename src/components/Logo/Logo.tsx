import React from 'react';

function Logo() {
  return (
    <>
      <img className="rounded-full" src="logo.png" alt="logo" />
      <span className="flex absolute h-5 w-5 top-0 md:top-2 right-1 md:right-2 -mt-1 -mr-1">
        <span className="relative inline-flex rounded-full h-4 w-4 bg-green-500"></span>
      </span>
    </>
  )
}

export default Logo;