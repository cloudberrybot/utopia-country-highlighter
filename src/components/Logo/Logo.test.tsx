import Logo from './Logo';
import { render } from '@testing-library/react';

test('renders correctly', () => {
  const { asFragment } = render(
    <Logo />
  );

  expect(asFragment()).toMatchSnapshot();
});
