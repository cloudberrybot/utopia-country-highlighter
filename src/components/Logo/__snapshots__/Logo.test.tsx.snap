// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <img
    alt="logo"
    class="rounded-full"
    src="logo.png"
  />
  <span
    class="flex absolute h-5 w-5 top-0 md:top-2 right-1 md:right-2 -mt-1 -mr-1"
  >
    <span
      class="relative inline-flex rounded-full h-4 w-4 bg-green-500"
    />
  </span>
</DocumentFragment>
`;
